### PR TITLE
Plz explain

### PR DESCRIPTION
**Pull request by apple502j**
I need sleep, this PR is very WIP

### CODE REVIEW

File: mappings/net/minecraft/util/registry/Registry.mapping [hunk 1]
```diff
@@ -216,3 +224,33 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 		ARG 1 key
 		ARG 2 entry
 	METHOD method_39673 getCodec ()Lcom/mojang/serialization/Codec;
+	METHOD method_40264 getTagsHolder (Lnet/minecraft/class_5321;)Ljava/util/Optional;
+		ARG 1 key
+	METHOD method_40265 getTagsHolder (I)Ljava/util/Optional;
+		ARG 1 rawId
+	METHOD method_40268 getOrCreateTagsHolder (Lnet/minecraft/class_5321;)Lnet/minecraft/class_6880;
+		ARG 1 key
+	METHOD method_40269 createTagsHolder (Ljava/lang/Object;)Lnet/minecraft/class_6880$class_6883;
```
Comment (apple502j) [inline mappings/net/minecraft/util/registry/Registry.mapping]: This is `getOrCreate`, fixing

File: mappings/net/minecraft/tag/TagsHolder.mapping [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		ARG 0 value
+	METHOD method_40224 testHolder (Ljava/util/function/Predicate;)Z
+		ARG 1 predicate
+	METHOD method_40225 isHeldBy (Lnet/minecraft/class_5321;)Z
+		ARG 1 holderRegistryKey
+	METHOD method_40226 areHolderIdsEqual (Lnet/minecraft/class_2960;)Z
+		ARG 1 id
+	METHOD method_40227 isBootstrapped ()Z
+	METHOD method_40228 stream ()Ljava/util/stream/Stream;
+	METHOD method_40229 toEither ()Lcom/mojang/datafixers/util/Either;
```
Comment (apple502j) [inline mappings/net/minecraft/tag/TagsHolder.mapping:16]: Probably incorrect

Comment (liach) [reply mappings/net/minecraft/tag/TagsHolder.mapping:16]: Hmm, I recall this looked like a holder of registry element than tag related... might need to look at it again later

File: mappings/net/minecraft/tag/TagsHolder.mapping [hunk 3]
```diff
@@ -0,0 +1,49 @@
+CLASS net/minecraft/class_6880 net/minecraft/tag/TagsHolder
+	METHOD method_40220 contains (Lnet/minecraft/class_6862;)Z
+		ARG 1 tagKey
+	METHOD method_40222 areRegistriesEqual (Lnet/minecraft/class_2378;)Z
+		ARG 1 registry
+	METHOD method_40223 createDirect (Ljava/lang/Object;)Lnet/minecraft/class_6880;
+		ARG 0 value
+	METHOD method_40224 testHolder (Ljava/util/function/Predicate;)Z
+		ARG 1 predicate
+	METHOD method_40225 isHeldBy (Lnet/minecraft/class_5321;)Z
```
Comment (apple502j) [inline mappings/net/minecraft/tag/TagsHolder.mapping:10]: Incorrect.

**Pull request closed without merging**